README.md

```diff
@@ -29,7 +29,7 @@ To compile thinkfan, you will need to have the following things installed:
 
 2. Then configure your build, either interactively:
    ```bash
-   ccmake ..
```

Comment:
Got it. I wasn't very familiar with `cmake` to begin with, and had no idea that there was a curses interface for it. I was confused when I tried running `ccmake` and the command failed, since that package isn't referred to anywhere else in the docs.